sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.tests import SavepointCase
+
+
+class TestSaleOrderLinePackagingQty(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.partner = cls.env.ref("base.res_partner_12")
+        cls.product = cls.env.ref("product.product_product_9")
+        cls.packaging = cls.env["product.packaging"].create(
+            {
+                "name": "Test packaging",
+                "product_id": cls.product.id,
+                "qty": 5.0
+            }
+        )
+
+    def test_product_packaging_qty(self):
+        order = self.env["sale.order"].create({"partner_id": self.partner.id})
+        order_line = self.env["sale.order.line"].create(
+            {
+                "order_id": order.id,
+                "product_id": self.product.id,
+                "product_uom": self.product.uom_id.id,
+                "product_uom_qty": 3.0
+            }
+        )
+        order_line.write({"product_packaging": self.packaging})
+        self.assertEqual(order_line.product_uom_qty, 5.0)
+        self.assertEqual(order_line.product_packaging_qty, 1.0)
+        order_line.write({"product_packaging_qty": 3.0})
```

Comment:
```suggestion
        order_line.write({"product_packaging_qty": 3.0})
        order_line._onchange_product_packaging_qty()
```

Reply:
Actually we don't need this onchange as it must be handled by the inverse function.

Reply:
We do need it because the inverse is called only when the record is saved. I'll add it back

Reply:
My bad :facepalm: , got tricked by the write in unit test and removed it. Thanks for restoring.